acq4/modules/Imager/Imager.py

```diff
@@ -330,6 +330,10 @@ def __init__(self, manager, name, config):
         # to select from [(dev1, channel1), ...]
         self.detectors = config.get('detectors', [config.get('detector')])
         
+        self.filterwheel = self.manager.getDevice(config['filterwheel'])
```

Comment:
1. Looks like this will fail if 'filterwheel' is not present in the config.
2. Maybe use 'filter' or 'filterDevice' instead of 'filterwheel', since we might like to support other types of filter-switching devices..